backend-rust/src/migrations/m0001-initialize.sql

```diff
@@ -0,0 +1,782 @@
+CREATE EXTENSION IF NOT EXISTS pg_trgm;
```

Comment:
Why do we have this file? It seems to be pretty much the same as what is already there.

I assume it at the very least means we have to remove the old ones?

Reply:
Yes, the old one needs to be removed and I will do that right before merging.
Merging in changes to the old file from main, is just simpler when I keep this around.